flox/core.py

```diff
@@ -1381,9 +1389,9 @@ def dask_groupby_agg(
         inds[-by.ndim :],
         concatenate=False,
         dtype=array.dtype,  # this is purely for show
-        meta=array._meta,
+        # meta=array._meta,
         align_arrays=False,
-        name=f"{name}-chunk-{token}",
+        # name=f"{name}-chunk-{token}",
```

Comment:
`_meta` and `name` are dask-specific. Are they used for anything important here or just for labelling tasks in the graph?

Reply:
if you don't provide meta, it will try to figure it out and then break?